traced/traced.go

```diff
@@ -218,6 +229,47 @@ func (tc *TraceCollector) collectWorker() {
 	}
 }
 
+func (tc *TraceCollector) writeJsonTrace(trace, name string) {
+	// open the trace, read it and transcode to json
+	in, err := os.Open(trace)
+	if err != nil {
+		panic(err)
+	}
+	defer in.Close()
+
+	gzipR, err := gzip.NewReader(in)
+	if err != nil {
+		panic(err)
+	}
+	defer gzipR.Close()
+
+	out, err := os.OpenFile(fmt.Sprintf("%s/%s.json", tc.jsonTrace, name), os.O_WRONLY|os.O_CREATE, 0644)
+	if err != nil {
+		panic(err)
+	}
+	defer out.Close()
+
+	var evt pb.TraceEvent
+	pbr := ggio.NewDelimitedReader(gzipR, 1<<20)
+	enc := json.NewEncoder(out)
```

Comment:
We need to use https://godoc.org/google.golang.org/protobuf/encoding/protojson

>  This package produces a different output than the standard "encoding/json" package, which does not operate correctly on protocol buffer messages.

Reply:
follow up from trying to use that -- turns out it doesn't work with proto2 protobufs, so we'll have to punt on this.